variables.tf

```diff
@@ -215,3 +215,9 @@ variable "cluster_name_random_string" {
   description = "Add a random string to the cluster name"
   default     = false
 }
+
+variable "internal_networks" {
+  description = "Subnet ranges for all internal networks"
+  type        = "list"
+  default     = []
+}
```

Comment:
@geekbass and @fatz with this addition of the `internal_networks` will this no longer require multiple `terraform apply`?

Reply:
indeed